infra/k8s/README.md

```diff
@@ -1,39 +1,97 @@
-# Running Testground with Kubernetes on Digitial Ocean backend
+# Setting up a self-managed Kubernetes cluster with kops on AWS for Testground
+
+In this directory, you will find:
+
+```
+» tree
+.
+├── README.md
+└── kops-weave                     # Kubernetes resources for setting up networking with Weave and Flannel
+```
+
+## Introduction
+
+Kubernetes Operations (kops) is a tool which helps to create, destroy, upgrade and maintain production-grade Kubernetes clusters from the command line. We use it to create a k8s cluster on AWS.
+
+We use CoreOS Flannel for networking on Kubernetes - for the default Kubernetes network
```

Comment:
Let's talk about this in terms of testground. I.e., which network do we use for the control network, which for the data network, etc.